File: NextNumber/NextNumber.py

# Implement the below functions
def next_greater(num):
    return -1

def next_smaller(num):
    return -1

def main():
    n = int(input())
    greater_number = next_greater(n)
    smaller_number = next_smaller(n)
    print(greater_number)
    if smaller_number == -1:
        print("No Number")
    else:
        print(smaller_number)


if __name__ == "__main__":
    main()
    

File: NextNumber/NextNumber.java
import java.io.*;
import java.util.*;

class NextNumber {
    // Complete the below functions
    public long nextGreater(long n) {

        return 0;
    }

    public long nextSmaller(long n) {

        return 0;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long n = scanner.nextLong();
        scanner.close();
        long greater = new NextNumber().nextGreater(n);
        long smaller = new NextNumber().nextSmaller(n);
        System.out.println(greater);
	if (smaller == -1) {
            System.out.println("No Number");
	}
        System.out.println(smaller);

    }
}
